src/components/ExportLicenseAsCSVModal/ExportLicenseAsCSVModal.js

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { mapValues, pickBy } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+import {
+  Button,
+  Checkbox,
+  Layout,
+  Modal,
+  ModalFooter,
+} from '@folio/stripes/components';
+
+import css from './ExportLicenseAsCSVModal.css';
+
+export default class ExportLicenseAsCSVModal extends React.Component {
+  static propTypes = {
+    onCompareLicenseTerms: PropTypes.func,
+    onClose: PropTypes.func,
+    selectedLicenses: PropTypes.arrayOf(PropTypes.string),
+    terms: PropTypes.arrayOf(PropTypes.object),
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.licenseInformation = {
+      name: <FormattedMessage id="ui-licenses.prop.name" />,
+      startDate: <FormattedMessage id="ui-licenses.prop.startDate" />,
+      endDate: <FormattedMessage id="ui-licenses.prop.endDate" />,
+      status: <FormattedMessage id="ui-licenses.prop.status" />,
+      type: <FormattedMessage id="ui-licenses.prop.type" />,
+    };
+
+    this.terms = {
+      value: <FormattedMessage id="ui-licenses.prop.termValue" />,
+      note: <FormattedMessage id="ui-licenses.prop.internalNote" />,
+      publicNote: <FormattedMessage id="ui-licenses.prop.publicNote" />,
+      internal: <FormattedMessage id="ui-licenses.prop.termVisibility" />,
+    };
+
+    this.termNames = props.terms.reduce((map, term) => {
```

Comment:
[These should probably be in alphabetical order of the `label`, right?](https://issues.folio.org/secure/attachment/24872/screenshot-2.png) (In which case, see above comment about object ordering).